compiler/record-minmax/include/RecordMinMax.h

```diff
@@ -51,9 +56,17 @@ class RecordMinMax
   void saveModel(const std::string &output_model_path);
 
 private:
+  void initializeCircleModule(const std::string &input_model_path);
+
+  std::vector<std::vector<std::vector<char>>> importH5Data(const std::string &input_data_path);
+
   std::unique_ptr<luci::Module> _module;
   std::unique_ptr<luci_interpreter::Interpreter> _interpreter;
   std::unique_ptr<MinMaxObserver> _observer;
+
+  // Used in parallel execution
+  std::vector<std::unique_ptr<luci_interpreter::Interpreter>> _vector_interpreters;
+  std::vector<std::unique_ptr<MinMaxObserver>> _vector_observers;
```

Comment:
How about name them without `vector` prefix? The data type already let us know the information.
```suggestion
  std::vector<std::unique_ptr<luci_interpreter::Interpreter>> _interpreters;
  std::vector<std::unique_ptr<MinMaxObserver>> _observers;
```